app/views/govuk_publishing_components/components/docs/devolved_nations.yml

```diff
@@ -0,0 +1,66 @@
+name: Devolved Nations (experimental)
+description: A banner for linking to guidance for other nations
+body: |
+  The component replaces uses of the important metadata component for guidance for other nations
+
+  The component can display:
+
+  * nations that the guidance relates to
+  * a list of alternative URLs where applicable
+shared_accessibility_criteria:
+  - link
+accessibility_criteria: |
+  - The list of countries are being read out without providing any context that individual guidance is available
```

Comment:
I'm not sure this is an accessibility criteria - it sounds more like the problem you're trying to solve? Criteria should be specific accessibility requirements that the component should meet. In this case I think you'd want something like:

```
The component must:

- provide context that guidance is available for other countries
```